### ISSUE TITLE
Binary: Main class missing error if the binary is located in a directory that has spaces

### ISSUE THREAD

**Issue by callmekungfu** [Cornutum/tcases#116]
When I tried to run the binary in my desktop directory, I got the error

```
Error: Could not find or load main class org.cornutum.tcases.TcasesCommand
```

Suspecting that this is likely due to the Java class path not being set properly, I debugged the `.bat` file and found it not properly concatenating the `.jar` directories for the class path. I moved the binary to a different directory with no spaces and it works.

Example of directory with spaces
```
C:\Users\Yonglin Wang\Desktop\software\tcases-3.3.0
```

This issue happened on both a Mac and Windows devices. Please update the bat files to handle directory names with spaces or at least mention it in the documentation.

**Comment by kerrykimbrough** [comment]
@callmekungfu, thanks for reporting this issue. I've merged PR #117 to fix the problem for Unix shells and I've verified it on my Mac.

Unfortunately, I no longer have access to a Windows system. Can you submit a PR with the necessary changes for that environment?

**Comment by callmekungfu** [comment]
Sure thing, I'll work on it over the course of the weekend.

**Comment by kerrykimbrough** [comment]
@callmekungfu, I realize you just installed 3.3.0, but just FYI, I recently released 3.4.0
